docs/app/pages/components/components-sections/buttons/floating-action-button-ng1/floating-action-button-ng1.component.html

```diff
@@ -42,13 +42,22 @@ <h4>Floating Action Button - Horizontal (Left)</h4>
 
 <uxd-api-properties>
     <tr uxd-api-property name="items" type="array" binding="variable" [required]="true">
-      Array of icons and events, that defines the child buttons shown when the primary button is expanded.
-      Each element of the array should contain an <code>icon</code> property which should be the name of an icon from the icon set.
-      There should also be an <code>event</code> property which should reference a function that will be called when a button is clicked.
-      <br>
-      Additionally, each item can have a <code>tooltip</code> property which, if defined, will display a tooltip when the button is hovered. 
-      The tooltip position can also be specified by adding a <code>tooltipPlacement</code> property with a value 
-      of <code>top</code>, <code>right</code>, <code>bottom</code> or <code>left</code>.
+      <p>
```

Comment:
Sorry for the misunderstanding. Something like:

An array of objects with the following properties:
* icon - ...
* event - ...
* tooltip - ...
* tooltipPlacement - ...

I think this will clearer.

Reply:
Ah sorry, my mistake, will fix this now

Reply:
Documentation should now be in the correct format